Wikipedia/UI-V5/WMFAppViewController.m

```diff
@@ -362,4 +360,14 @@ - (void)searchLanguageDidChangeWithNotification:(NSNotification*)note {
     [self configureSearchViewController];
 }
 
+#pragma mark - UINavigationControllerDelegate
+
+- (void)navigationController:(UINavigationController*)navigationController
+      willShowViewController:(UIViewController*)viewController
+                    animated:(BOOL)animated {
+    BOOL isToolbarEmpty = [viewController toolbarItems].count == 0;
+    [self.rootTabBarController wmf_setTabBarVisible:isToolbarEmpty animated:NO completion:nil];
```

Comment:
Also the tab bar is now being hidden manually instead of using the API that was created for it - hidesBottomBarWhenPushed